ENGAGEHF/Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,75 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+//  DashboardView.swift
+//  ENGAGEHF
+//
+//  Created by Nick Riedman on 4/3/24.
+//
+
+import SpeziAccount
+import SpeziMockWebService
+import SwiftUI
+
+struct Greeting: View {
+    @State private var dateString: String?
+    
+    var body: some View {
+        HStack {
```

Comment:
I would suggest follow the screenshots in the document to figure out the functionality but not for any specific design guidelines. E.g. this first header looks a bit strange IMO but we can keep it there for now.

E.g. this would be a slight improvement, please check out the usage of the different font styles (e.g. bold), foregroundStyle for the color, and the top alignment in the HStack to ensure that the text is allied to the top of the stack:
```swift
        HStack(alignment: .top) {
            Text("Hello, world!")
                .font(.title.bold())
                .accessibilityLabel(Text("DASHBOARD_GREETING"))
            Spacer()
            Text(.now, style: .date)
                .font(.title3)
                .foregroundStyle(.secondary)
                .accessibilityLabel(Text("DASHBOARD_DATE"))
        }
```